src/analysis/analysis.py

```diff
@@ -138,6 +124,35 @@ def main(data_filepath, out_filepath):
         column='Model:N'
     ).save(os.path.join(out_filepath,'model_performance.png'))
 
+def train_random_forest(X_train, y_train, preprocessor, scoring_metrics, cross_val_results):
```

Comment:
We should have docstrings for each of these funcions.

Reply:
done